plugins/modules/k8s_rollback.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2020, Julien Huon <@julienhuon> Institut National de l'Audiovisuel
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
```

Comment:
Line 12 to 14 is no longer required and can be removed. Please see https://github.com/ansible-collections/overview/issues/45#issuecomment-627922948